tests/unit/reqstool/model_generators/test_testdata_model_generator.py

```diff
@@ -0,0 +1,40 @@
+# Copyright © LFV
+
+import re
+
+import pytest
+
+from reqstool.model_generators.testdata_model_generator import TestDataModelGenerator
+
+karate_method_names = [
+    "[1.4:55] Create a subscripiton with filter and receive messages",
+    "[1:38] Create a subscripiton with filter and receive messages",
+    "[1.2:53] Create a subscripiton with filter and receive messages",
+]
+
+unit_method_names = [
+    "testFlightIdAircraftId",
+    "testFlightIdAircraftId(String, int)[1]",
+]
+
+
+@pytest.mark.parametrize("method_name", karate_method_names)
+def test_karate_method_identifier_regex(method_name):
+    karate_match = re.match(TestDataModelGenerator.KARATE_METHOD_IDENTIFIER_REGEX, method_name)
+    assert karate_match is not None
+    assert karate_match.group(1) == "Create a subscripiton with filter and receive messages"
+
+
+@pytest.mark.parametrize("method_name", unit_method_names)
+def test_unit_method_identifier_regex(method_name):
+    unit_match = re.match(TestDataModelGenerator.UNIT_METHOD_IDENTIFIER_REGEX, method_name)
+    assert unit_match is not None
+    assert unit_match.group(1) == "testFlightIdAircraftId"
+
+
+@pytest.mark.skip(reason="Awaiting testdata update")
```

Comment:
This should be completed. Let's open a separate issue for that?

Reply:
Ok. Once code and CI/CD has been migrated here we'll also move over existing tickets.